core/src/js/components/overlays/turntimer/turn-timer-widget.component.ts

```diff
@@ -93,7 +93,7 @@ export class TurnTimerWidgetComponent extends AbstractSubscriptionComponent impl
 		this.player$ = combineLatest(
 			interval(1000),
 			this.store.listenDeckState$(
-				(state) => state?.playerDeck?.hero?.playerName,
+				(state) => this.i18n.translateString('turn-timer.player'),
```

Comment:
I think this should be driven by a (new) setting. I can totally see players who use the Streamer mode not wanting to have their names displayed, but on the other hand I think it's nice to view the names instead of just "you" / "opponent".

Reply:
It seems to me so easier to perceive information at a glance.

Reply:
Yeah it's probably a matter of taste, which is why I think having a setting could make sense